### PR TITLE
Hao eliminate popups

### LINKED ISSUE #11: FEATURE: Eliminate purchase notification popups
# User Story

* As a User (especially mobile)
* I can Buy cows without seeing toast message pop up.
* So that I can buy cows more easily

### Discussion

Right now when buying many cows by spam-clicking, the screen floods with times "Cow bought!" popups:

image

This is especially distracting on mobile, as the popups block the interface and buttons.

These toast message aren't adding any value for the user, so we should just eliminate them in the case where the
cow purchase was successful.  The messages may still be useful if/when there is an error condition, but if the purchase
is successful, it will be reflected in the cow count and dollar amounts on the screen.

(The toast message is a leftover feature from before those numbers were being reliably displayed and updated... it was useful at the time, but isn't anymore.)

### Acceptance Criteria

- [ ] No Toast message is generated when purchasing cows.

### PR DESCRIPTION
**Pull request by haowu0521**
In this PR, I eliminated the popups when buying a cow and modified corresponding test in playpage test.

Closes #11

### CODE REVIEW

File: frontend/src/main/pages/LeaderboardPage.js
Comment (EthanSun210) [inline frontend/src/main/pages/LeaderboardPage.js:1]: This file adds a "Back" button for the leaderboard page.

Closes: #12

File: frontend/src/main/pages/PlayPage.js
Comment (EthanSun210) [inline frontend/src/main/pages/PlayPage.js:1]: Do we also need to eliminate the popups for "Cow sold!"

File: frontend/src/main/pages/LeaderboardPage.js
Comment (chriscerie) [reply frontend/src/main/pages/LeaderboardPage.js:1]: Why is this here? It's a completely different feature and you already have a PR for it. It's now more difficult to reconcile changes if any one of these PRs change.

**Review by chriscerie — CHANGES_REQUESTED** [review]

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**